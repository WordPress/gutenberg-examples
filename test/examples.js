/**
 * External dependencies
 */
import { create as createTestRenderer } from 'react-test-renderer';

/**
 * WordPress dependencies
 */
import { Button } from '@wordpress/components';

// @see https://jestjs.io/docs/en/getting-started
test( 'adds 1 + 2 to equal 3', () => {
	function sum( a, b ) {
		return a + b;
	}

	expect( sum( 1, 2 ) ).toBe( 3 );
} );

// @see https://jestjs.io/docs/en/tutorial-react
test( 'renders Button from WordPress components', () => {
	const testRenderer = createTestRenderer( <Button>Click Me!</Button> );

	expect( testRenderer.toJSON() ).toMatchSnapshot();
} );
