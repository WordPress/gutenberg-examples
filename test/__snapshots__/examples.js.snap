// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Button from WordPress components 1`] = `
<button
  className="components-button"
  type="button"
>
  Click Me!
</button>
`;
